### PR TITLE
get user emmbedded fetaure

### PR DESCRIPTION
**Pull request by Teppei-Kanayama**
Purpose:
Extract user embedded vector using trained gcmc model

Problems to be fixed in this code: 
Function ```get_user_feature_with_new_items()``` requires additional_dataset, which is essentially unnecessary to extract user feature.

### CODE REVIEW

**Comment by nishiba** [comment]
LGTM

File: redshells/model/graph_convolutional_matrix_completion.py [hunk 1]
```diff
@@ -314,6 +319,26 @@ def _predict(cls, user_ids: List, item_ids: List, with_user_embedding, graph: Gr
...
+    def _get_user_feature(cls, user_ids: List, item_ids: List, with_user_embedding,
+                          graph: GraphConvolutionalMatrixCompletionGraph, dataset: GcmcGraphDataset,
+                          session: tf.Session) -> np.ndarray:
+        if graph is None:
+            RuntimeError('Please call fit first.')
+
+        rating_adjacency_matrix = dataset.train_rating_adjacency_matrix()
+        user_indices, item_indices = dataset.to_indices(user_ids, item_ids)
+        if not with_user_embedding:
+            user_indices = np.array([0] * len(user_indices))  # TODO use default user index.
```
Comment (hirosassa) [inline redshells/model/graph_convolutional_matrix_completion.py]: `TODO` is remaining. Do you fix this in this PR？ or you can post it to issue and fix it later

**Comment by 6syun9** [comment]
LGTM

File: redshells/model/graph_convolutional_matrix_completion.py (hunk 1 above)
Comment (Teppei-Kanayama) [reply redshells/model/graph_convolutional_matrix_completion.py]: I've just created the issue ticket about this TODO.

**Comment by Teppei-Kanayama** [comment]
@hirosassa
I added unit test for this Pull request. Please review again.

**Comment by hirosassa** [comment]
LGTM

**Pull request merged**